lib/view_model/after_auth_view_models/profile_view_models/edit_profile_view_model.dart

```diff
@@ -61,6 +69,147 @@ class EditProfilePageViewModel extends BaseModel {
     }
   }
 
+  /// modal sheet to choose image.
+  ///
+  /// more_info_if_required
+  ///
+  /// **params**:
+  /// * `context`: define_the_param
+  ///
+  /// **returns**:
+  ///   None
+  void showImagePickerIcons(BuildContext context) {
```

Comment:
If it is an ImagePicker Dialog, then why is it named showImagePickerIcons ?
Moreover, passing around contexts in functions is not the standard way to go about this. Please have a look at MVVM architecture once.